Correct errors in network-broker.md (#1419)
semgrep/semgrep-docs: docs/semgrep-ci/network-broker.md

Ensure that you are logged in to the server where you want to run Semgrep Network Broker. Complete the following steps while logged in to that server.

> 1. Create a `config.yaml` file similar to the following snippet. The steps required to generate values for the placeholders `SEMGREP_LOCAL_ADDRESS`, `YOUR_PRIVATE_KEY`, `YOUR_BASE_URL`, `YOUR_TOKEN`, are provided in subsequent steps of this guide.

Reviewer: ```suggestion
1. Create a `config.yaml` file similar to the following snippet. The steps required to generate values for the placeholders `SEMGREP_LOCAL_ADDRESS`, `YOUR_PRIVATE_KEY`, `YOUR_BASE_URL`, and `YOUR_TOKEN`, are provided in subsequent steps of this guide.
```

  ```

1. The broker requires a WireGuard keypair to establish a secure connection. To generate your private key `YOUR_PRIVATE_KEY`:

   1. Determine the [network broker version](https://github.com/semgrep/semgrep-network-broker/pkgs/container/semgrep-network-broker) you want to use. The format should be similar to `v0.14.0`.

   1. Run the following command in the CLI to generate your private key, replacing the placeholder with the network broker version number:
  <pre class="language-console"><code>docker run ghcr.io/semgrep/semgrep-network-broker:<span className="placeholder">VERSION_NUMBER</span> genkey</code></pre>

1. Run the following command in the CLI to generate your public key, replacing the placeholders with your private key generated in the previous step and the network broker version number:

  <pre class="language-console"><code>echo `<span className="placeholder">YOUR_PRIVATE_KEY</span>` | sudo docker run -i ghcr.io/semgrep/semgrep-network-broker:<span className="placeholder">VERSION_NUMBER</span> pubkey</code></pre>

  :::info Key sharing 
  Your public key is safe to share. Do **not** share your private key with anyone, including Semgrep.
  :::

> 1. Update the `config.yaml` file with your private key generated in step 2:

Reviewer 2: ```suggestion
1. Update the `config.yaml` file with your private key generated in a previous step:
```

I think we should make this vague in case we update steps in the future

Author: Updated by removing reference to previous step

> 1. Update the `config.yaml` file with the private key generated in step two:

Reviewer: ```suggestion
1. Update the `config.yaml` file with the private key generated in step 2:
```

      ...
  ```

> 1. Add your public key generated in step 3 to the Semgrep Cloud Platform:

Reviewer 2: ```suggestion
1. Add the public key that you generated in a previous step to Semgrep Cloud Platform:
```

Author: Updated by removing reference to previous step.

> 1. Add the public key generated in step three to the Semgrep Cloud Platform:

Reviewer: ```suggestion
1. Add the public key generated in step 3 to the Semgrep Cloud Platform:
```

   1. Log in to Semgrep Cloud Platform.
   2. Navigate to **Settings** > **Broker**.

>    3. Paste `YOUR_PUBLIC_KEY` and click **Add Public Key**.

Reviewer: This seems even more confusing to me. You don't want to paste the string `YOUR_PUBLIC_KEY`, presumably. Do we want to say something like "Paste the generated public key" or "Paste the content of your public key" maybe?

Author: "Paste the generated public key" to keep it consistent with the term used on line 83.

Author: After going through the doc again, the `YOUR_PUBLIC_KEY` variable is not helpful and is now removed.

I altered all instances of 'the private key' and 'the public key' to be 'your private key' and 'your public key' to keep things consistent and to prevent confusion with Semgrep's private key.
